sourcemash/frontend/templates/subscribe-modal.tpl

```diff
@@ -0,0 +1,8 @@
+<div class="modal-content">
+  <h4>Interested?</h4>
+  <p>You <span id="unsubscribed-source-title"></span> "<span id="unsubscribed-item-title"></span>" even though you're not subscribed to <i><span id="unsubscribed-feed-title"></span></i>. Would you like to add this feed to your subscriptions?<p>
```

Comment:
Do we have quotation issues here? Why is the span for unsubscribed-item-title in quotes?